test_scripts/Polices/External_UCS/External_Consent_ON/011_ATF_Policies_External_Consent_ON_through_ignition_cycles_allowed_RPCs.lua

```diff
@@ -0,0 +1,173 @@
+  Test[TEST_NAME_ON .. "MainCheck_RPC_of_Group001_is_allowed"] = function(self)
+    local corid = self.mobileSession:SendRPC("SubscribeVehicleData", {rpm = true})
+    EXPECT_HMICALL("VehicleInfo.SubscribeVehicleData")
+    :Do(function(_,data)
+        self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS",{})
+      end)
+    EXPECT_RESPONSE(corid, {success = true , resultCode = "SUCCESS"})
+    EXPECT_NOTIFICATION("OnHashChange")
+  end
+end
+
+local function IgnitionOffOnActivateApp(test_case_name)
+  common_steps:IgnitionOff("Precondition_Ignition_Off_" .. test_case_name)
+  common_steps:IgnitionOn("Precondition_Ignition_On_" .. test_case_name)
+  common_steps:AddMobileSession("Precondition_Add_Mobile_Session_1_" .. test_case_name, "mobileConnection", "mobileSession")
+  common_steps:RegisterApplication("Precondition_Register_Application_1_" .. test_case_name, "mobileSession", config.application1.registerAppInterfaceParams)
+  common_steps:ActivateApplication("Precondition_Activate_Application_1_" .. test_case_name, config.application1.registerAppInterfaceParams.appName)
+  Test["Precondition_Wait_For_Database_After_Activate_Application_1_" .. test_case_name] = function(self)
+    common_functions:DelayedExp(5000)
+  end
+end
+
+------------------------------------------------------------------------------------------------------
+------------------------------------------ Tests -----------------------------------------------------
+------------------------------------------------------------------------------------------------------
+--------------------------------------------------------------------------
+-- TEST 10:
+-- In case
+-- SDL has received SDL.OnAppPermissionConsent ("externalConsentStatus: ON") from HMI
+-- SDL must
+-- use this value through ignition cycles
+-- until this externalConsentStatus value is changed by corresponding notification from HMI.
+--------------------------------------------------------------------------
+-- Test 10.01:
+-- Description:
+-- 1. disallowed_by_external_consent_entities_off.
+-- 2. HMI->SDL: OnAppPermissionConsent(externalConsentStatus ON).
+-- 3. Ignition Off then On.
+-- 4. Expected Result: externalConsentStatus is kept. RPC is allowed
+
+--------------------------------------------------------------------------
+-- Precondition:
+-- Prepare JSON file with consent groups. Add all consent group names into app_polices of applications
+-- Request Policy Table Update.
+--------------------------------------------------------------------------
+Test[TEST_NAME_ON.."Precondition_Update_Policy_Table"] = function(self)
+  -- create json for PTU from sdl_preloaded_pt.json
+  local data = common_functions_external_consent:ConvertPreloadedToJson()
+  -- insert Group001 into "functional_groupings"
+  data.policy_table.functional_groupings.Group001 = {
+    user_consent_prompt = "ConsentGroup001",
+    disallowed_by_external_consent_entities_off = {{
```

Comment:
@istoimenova,  Please find update in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1335